lib/limber/helper.rb

```diff
@@ -166,12 +166,17 @@ def each_submission_template
         sequencing_request_types.each do |sequencing_request_type|
           next if SubmissionTemplate.exists?(name: name_for(cherrypick, sequencing_request_type))
 
+          # Hiseq templates are hidden as of Y24-181 so we set them as superceeded by unknown
+          # This allows us to keep the template in case it needs to be restored but not make it visible
+          superceded_by_id = sequencing_request_type&.name&.downcase&.include?('hiseq') ? -2 : -1
```

Comment:
Why is codecov whining about tests here and further below when you have a test?

Reply:
ah, because the test is for the rake task rather than this class maybe. Add some small unit tests just to check obvious use cases?

Reply:
Yeah just added some really basic tests for this as there were none previously.